config.json

```diff
@@ -12,14 +12,24 @@
   "version": 3,
   "online_editor": {
     "indent_style": "space",
-    "indent_size": 2
+    "indent_size": 2,
+    "ace_editor_language": "typescript",
```

Comment:
```suggestion
    "ace_editor_language": "javascript",
```

@ErikSchierboom I don't know if this will actually work, but there is no `javascript` folder.

Reply:
@SleeplessByte Ah yes, but there _is_ a `typescript` file: https://github.com/ajaxorg/ace/blob/master/lib/ace/mode/typescript.js

> See the full list of identifiers **(filenames, not directories)**

Emphasis mine :)

Reply:
Oh. I read that in reverse.

Reply:
:)